website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -24,15 +24,15 @@ We encourage you to include the A in your DEI efforts: **Diversity + Equity + In
 
 #### Disabilities coordinator
 
-We recommend appointing a designated Disabilities coordinator to handle any prompt communications involving accommodations before and during the conference. This includes communication with attendees, speakers, facilities, and other event organizers. For signing attendees specifically, the coordinator typically:
-- Facilitates communications between interpreters, assigns them to deaf person(s) and location, solicits feedback, and determines fit (e.g., some deaf people may not like the interpreter).
+We recommend appointing a designated disabilities coordinator to handle any prompt communications involving accommodations before and during the conference. This includes communication with attendees, speakers, facilities, and other event organizers. For signing attendees specifically, the coordinator typically:
+- Facilitates communications between interpreters, assigns them to deaf attendees and location, solicits feedback, and determines fit (e.g., some deaf people may not like the interpreter).
```

Comment:
- Facilitates communications between interpreters, assigns them to deaf attendees and location, solicits feedback, and determines fit (e.g., some deaf people may not like the interpreter's signing style, clarity, or quality).

Reply:
Good point!